README.md

```diff
@@ -204,6 +204,37 @@ Manifest is loaded as **template**, so you can use variables, Go **range** and *
             "header2": "value"
         },
 
+        // Cookies can be added to the request
+        "cookies": {
+            // name of a cookie to be set
+            "cookie1": {
+                // A cookie can be get parsed from store if it was saved before
+                // It will ignore the cookie if it is not set
+                "value_from_store": "sess_cookie",
+                // Or its values can be directly set, overriding the one from store, if defined
+                "value": "value"
+            },
+            "cookie2": {
+                "value_from_store": "ads_cookie",
+            }
+        },
+
+        // Special headers `X-Test-Set-Cookie` can be populated in the request (on per entry)
+        // It is used in the builting `http_server` to automatically set those cookies on response
+        // So it is useful for mocking them for further testing
+        "x-test-set-cookie": [
+            {
+                "name": "sess_cookie",
+                "value": "myauthtoken"
+            },
+            {
+                "name": "jwtoken",
+                "value": "tokenized",
+                "Path": "/auth",
+                "HttpOnly": true
```

Comment:
Why is this capital here? 

http_only, path

Is this all what can be set or is there more fields?

Reply:
http.Cookie is not mapped to any json, no tags
So I'll have to create our own struct, otherwise its getting quite cumbersome
There are more fields, yes, should we add all of them?